client/dashboard/me/tax-details/use-data-form-country-codes.ts

```diff
@@ -0,0 +1,41 @@
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import useCountryList, { isVatSupported } from '../../app/hooks/use-country-list';
+import type { CountryListItem } from '../../data/types';
+
+interface CountryCodeOption {
+	label: string;
+	value: string;
+}
+
+function getUniqueCountries< C extends CountryListItem >( countries: C[] ): C[] {
+	const unique: C[] = [];
+	countries.forEach( ( country ) => {
+		if ( unique.map( ( x ) => x.code ).includes( country.code ) ) {
+			return;
+		}
+		unique.push( country );
+	} );
+	return unique;
+}
+
+export default function useDataFormCountryCodes(): CountryCodeOption[] {
```

Comment:
Maybe this is the sort of complex operation I would expect to be in `utils/tax.ts`

Reply:
Moved in 3b645b9.